pkg/engine/plan.go

```diff
@@ -223,6 +225,18 @@ func (res *planResult) Walk(ctx *Context, events deploy.Events, preview bool) (d
 				return
 			}
 
+			// Warn the user if they're not updating a resource whose initialization failed.
+			if step.Op() == deploy.OpSame && len(step.Old().InitErrors) > 0 {
+				prefix := colors.SpecImportant + "         [ERROR]" + colors.Reset
```

Comment:
it's a bit weird that we're mixing warning/[Error] here.  if this really is an error... then we should emit an error.  if it's not really an error, we should just do it as a warning (IMO) .

Reply:
So what we want to communicate is:

1. That there were initialization errors previously.
1. What those initialization errors were.
1. You don't _have_ to update that resource to fix them but you definitely _should_.

I think I'm ok with pretty much any text that communicates these things effectively. If you have suggestions, let me know... the screenshot below gives a flavor of what it looks like now.

Reply:
I mean... we're already printing out that there's a warning here.  So i might just say that there's no need for the [Error] text.   If anything, we can say [Issue] if we want to be generic about it and we want to enuemrate the issues that happened.

Note: i find this text very clunky "failed with the following errors, but this plan does not update it".

Reply:
```
warning: Previous initialization of this resource encountered problems.  Updating resource recommended to address this. 
         Issue #1: Yadda yadda 
         Issue #2: Yadda yadda
```

Reply:
Ok, that sounds good, we'll use that wording.

Reply:
Looks like this now:

<img width="657" alt="screen shot 2018-07-20 at 5 34 15 pm" src="https://user-images.githubusercontent.com/1409156/43030293-3ededfda-8c43-11e8-98ff-fae3f4a13f22.png">